src/components-examples/cdk-experimental/selection/cdk-selection-column/cdk-selection-column-example.css

```diff
@@ -0,0 +1,7 @@
+table {
```

Comment:
Could you add `.example-` CSS classes rather than targeting element selectors directly?